README.md

```diff
@@ -2,30 +2,287 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Practice creating a web application that connects to a data source using the
+  Spring Framework.
+- Connect an application to a PostgreSQL database.
+- Create a DTO and `@Entity` class.
+- Create a `@Repository` class.
```

Comment:
Consider changing 'class' to 'interface'

Reply:
Yes this is a typo. Thanks!